Microsoft.Azure.CosmosRepository/test/DefaultRepositoryFactoryTests.cs

```diff
@@ -17,15 +17,17 @@ public class DefaultRepositoryFactoryTests
         [Fact]
         public void RepositoryFactoryCorrectlyGetsRepositoryTest()
         {
-            IServiceCollection services = new ServiceCollection();
             IConfigurationRoot configuration =
                 new ConfigurationBuilder()
                     .AddInMemoryCollection(new Dictionary<string, string>
-                    {
-                        ["RepositoryOptions:CosmosConnectionString"] = "Testing"
-                    })
+                                           {
+                                               ["RepositoryOptions:CosmosConnectionString"] = "Testing"
+                                           })
```

Comment:
```suggestion
                    {
                        ["RepositoryOptions:CosmosConnectionString"] = "Testing"
                    })
```

Unnecessary styling change.